pouta_blueprints/rules.py

```diff
@@ -0,0 +1,110 @@
+from pouta_blueprints.models import Blueprint, BlueprintTemplate, Instance, GroupUserAssociation
+from pouta_blueprints.views.commons import is_group_manager
+from sqlalchemy import or_, and_
+from sqlalchemy.orm import load_only
+from sqlalchemy.sql.expression import true
+import itertools
+# import logging
+
+
+def apply_rules_blueprint_templates(user, args=None):
+
```

Comment:
This empty line is just screaming for a simple docstring. Doesn't have to be more than one sentence but that sentence reduces cognitive load of reader.